### DISCUSSION TITLE
App Set ID SDK is missing

### DISCUSSION THREAD

**Discussion by riffre** [admob-plus/admob-plus#591] [Fund Features]
Ad serving on my app is disabled as App Set ID is not implemented. Please provide an update. Also I can't implement mediation due to AndroidX version clashes. This library is using 1.6.+ while mediation requires 2.7+ for AndroidX.

Following is the link for App Set ID docs:

https://developer.android.com/training/articles/app-set-id

I tried implementing, but being not a java developer, could not succeed.

**Comment by ratson** [comment]
I think this is out of the scope of this project.
And a third-party plugin could implement it if needed.

If anyone could describe what the API should look like
and willing to fund this feature, I am open to work on it.
